### PR TITLE
Bash inst

### PR DESCRIPTION
**Pull request by Julian-Mentasti**
## Description
* Added a bash file structure creator

## Related Issue

#193 

## How Has This Been Tested? (Optional)

Me manually running the programme. 

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [x] New Algorithm (non-breaking change which adds functionality)

- [X] Documentation
## Checklist:

- [x] My code follows the code style of this project.

- [ ] My change requires a change to the documentation.

- [X] I have updated the documentation accordingly.

- [x] I have read the **CONTRIBUTING** document.

Maybe add documentation in the main readme

### CODE REVIEW

**Comment by Julian-Mentasti** [comment]
Sure, @tstreamDOTh @GnikDroy. Ill add preset languages, checking for existing directories, getting space to work and it will show some examples on how to name things.

**Comment by Julian-Mentasti** [comment]
@tstreamDOTh I just committed a new update, let me know if you want to change anything :D

File: runme.bh [hunk 1]
```diff
@@ -2,24 +2,64 @@
...
       exit
     fi
+    # Will capitalise everything after a space
+}
+
+# Asks the user that language they would like to use
+pickLanguague() {
+    echo "Select the language you want to implement "$1" in?"
+    echo "1)  C++ (.cpp)"
+    echo "2)  C++ (.c++)"
```
Comment (GnikDroy) [inline runme.bh]: Does anyone use this? For consistency let's only keep 1

Comment (Julian-Mentasti) [reply runme.bh]: Not sure. I usually use cpp, but if you look at codezila/Graphs both c++ and cpp is used.

Comment (GnikDroy) [reply runme.bh]: I meant the extension (.c++). Noone uses it I think. You are right about folders. We can select either C++ or Cpp. C++ seems more clear to me. What about you?

Comment (tstreamDOTh) [reply runme.bh]: Make it C++ (.cpp)
C++ is the folder name
.cpp is the extension

Comment (Julian-Mentasti) [reply runme.bh]: Sure, sounds good to me

**Comment by Julian-Mentasti** [comment]
Made a fresh PR, not sure why the other files where showing e168f74d826ba6acafe8bcda93cefb054674dd94

**Pull request closed without merging**